pnpm-lock.yaml

```diff
@@ -10000,7 +10000,7 @@ importers:
         version: link:../../test/test-drivers
       '@fluid-tools/benchmark':
         specifier: ^0.48.0
-        version: 0.48.0
+        version: link:../../../tools/benchmark
```

Comment:
You'll want to revert this before we merge.

Reply:
Removed the OpSize test here. It's used to test locally. I won't merge it with benchmark-tool custom option PR.